diff --git a/src/core/ui/about.cpp b/src/core/ui/about.cpp
index 33f2b15..6c47467 100644
--- a/src/core/ui/about.cpp
+++ b/src/core/ui/about.cpp
@@ -31,16 +31,8 @@ AboutDialog::AboutDialog(QWidget *parent):
     _ui->setupUi(this);
     _ui->labAppName->setText(_ui->labAppName->text() + QString(" <b>") + qApp->applicationVersion() + QString("</b>"));
 
-    QString versionInfo;
-    versionInfo = tr("built on ");
-    versionInfo.append(__DATE__);
-    versionInfo.append(" ");
-    versionInfo.append(__TIME__);
-
     _ui->labQtVer->setText(tr("using Qt ") + qVersion());
 
-    _ui->labVersion->setText(versionInfo);
-
     QTabBar *tabs = new QTabBar;
 
     _ui->frame->layout()->addWidget(tabs);
