Description: Disable cli upload option
Author: Alf Gaida <agaida@siduction.org>
Bug-Debian: https://bugs.debian.org/834517
Last-Update: 2016-09-19

--- a/src/core/core.cpp
+++ b/src/core/core.cpp
@@ -536,8 +536,9 @@
         if (_cmdLine.isSet(_screenTypeOpts.at(i)))
             _conf->setDefScreenshotType(i);
 
+/*
 #ifdef SG_EXT_UPLOADS
-    /// FIXMA - In module interface need add the mthod for geting module cmdLine options
+    /// FIXMA - In module interface need add the mthod for geting module cmdLine option
     const QString UPLOAD_CMD_PARAM = "upload";
     const QString UPLOAD_CMD_PARAM_SHORT = "u";
     QCommandLineOption u(QStringList() << UPLOAD_CMD_PARAM_SHORT << UPLOAD_CMD_PARAM);
@@ -549,6 +550,9 @@
     } else
         initWindow();
 #endif
+*/
+
+initWindow();
 
 }
 
