Description: Disable cli upload option
Author: Alf Gaida <agaida@siduction.org>
Bug-Debian: https://bugs.debian.org/834517
Last-Update: 2016-09-19

--- a/src/modules/uploader/moduleuploader.cpp
+++ b/src/modules/uploader/moduleuploader.cpp
@@ -36,11 +36,11 @@
 
 ModuleUploader::ModuleUploader(QObject *parent) :
     QObject(parent), _ignoreCmdParam(false),
-    _optUpload(QStringList() << UPLOAD_CMD_PARAM_SHORT << UPLOAD_CMD_PARAM)
+    _optUpload(QStringList() << UPLOAD_CMD_PARAM_SHORT << UPLOAD_CMD_PARAM) 
 {
     QString optUploadDescr = tr("Upload the screenshot to the default image host");
     _optUpload.setDescription(optUploadDescr);
-    Core::instance()->addCmdLineOption(_optUpload);
+    /* Core::instance()->addCmdLineOption(_optUpload); */
 }
 
 QString ModuleUploader::moduleName()
