Description: Disable cli upload option
Author: Alf Gaida <agaida@siduction.org>
Bug-Debian: https://bugs.debian.org/834517
Last-Update: 2016-09-19

--- a/src/modules/uploader/moduleuploader.cpp
+++ b/src/modules/uploader/moduleuploader.cpp
@@ -40,7 +40,7 @@ ModuleUploader::ModuleUploader(QObject *parent) :
 {
     QString optUploadDescr = tr("Upload the screenshot to the default image host");
     _optUpload.setDescription(optUploadDescr);
-    Core::instance()->addCmdLineOption(_optUpload);
+    // Core::instance()->addCmdLineOption(_optUpload);
 }
 
 QString ModuleUploader::moduleName()
