diff --git a/src/core/modulemanager.cpp b/src/core/modulemanager.cpp
index 8778e93..b35202c 100644
--- a/src/core/modulemanager.cpp
+++ b/src/core/modulemanager.cpp
@@ -18,9 +18,9 @@
 
 #include "modulemanager.h"
 
-#ifdef SG_EXT_UPLOADS
-#include "src/modules/uploader/moduleuploader.h"
-#endif
+// #ifdef SG_EXT_UPLOADS
+// #include "src/modules/uploader/moduleuploader.h"
+// #endif
 
 #ifdef SG_EXT_EDIT
 #include "src/modules/extedit/moduleextedit.h"
@@ -35,10 +35,10 @@ ModuleManager::ModuleManager()
 
 void ModuleManager::initModules()
 {
-#ifdef SG_EXT_UPLOADS
-    ModuleUploader *uploader = new ModuleUploader();
-    _modules->insert(MOD_UPLOADER , uploader);
-#endif
+//#ifdef SG_EXT_UPLOADS
+//    ModuleUploader *uploader = new ModuleUploader();
+//    _modules->insert(MOD_UPLOADER , uploader);
+//#endif
